test/jest/__mock__/stripesSmartComponents.mock.js

```diff
@@ -4,8 +4,18 @@ jest.mock('@folio/stripes/smart-components', () => ({
   ClipCopy: jest.fn(() => null),
   makeQueryFunction: jest.fn((value) => value),
   CheckboxFilter: jest.fn(() => null),
+  NoteCreatePage: (props) => {
```

Comment:
I do not see the usage of this component inside PickupServicePointFilter. Do we really need to mock it?

Reply:
I do not see the usage of this component inside PickupServicePointFilter. Do we really need to mock it?